source/operations/concepts/availability-and-resiliency.rst

```diff
@@ -0,0 +1,153 @@
+.. _minio_availability-resiliency:
+
+===========================
+Availability and Resiliency
+===========================
+
+.. default-domain:: minio
+
+.. contents:: Table of Contents
+   :local:
+   :depth: 2
+
+.. meta::
+   :keywords: availability, resiliency, healing, recovery, distributed
+   :description: Information on MinIO Availability and Resiliency features in production environments
+
+This page provides an overview of MinIO's availability and resiliency design and features from a production perspective.
+
+.. note::
+
+   The contents of this page are intended as a best-effort guide to understanding MinIO's intended design and philosophy behind availability and resiliency. 
+   It cannot replace the functionality of |subnet|, which allows for coordinating with MinIO Engineering when planning your MinIO deployments.
+
+   Community users can seek support on the `MinIO Community Slack <https://slack.min.io>`__. 
+   Community Support is best-effort only and has no SLAs around responsiveness.
+
+MinIO implements :ref:`erasure coding <minio-erasure-coding>` as the core component in providing availability and resiliency during drive or node-level failure events.
+   MinIO shards each object into data and :ref:`parity <minio-ec-parity>` blocks and distributes those blocks across a single :ref:`erasure set <minio-ec-erasure-set>`.
```

Comment:
Maybe `slices` instead of `blocks` to avoid unintentional association with block storage?